README.md

```diff
@@ -77,31 +86,52 @@ We could simply treat it as the tree
  "Hello!"     "Sup?"
 ```
 
-### Finding a Node
+## Practice exploring the DOM
+
+Remember when we said that we could organize our tree in such a way that a
+node's metadata didn't interfere with finding its children? It turns out that
+not only does providing additional information about a node make it more
+_useful_, it also makes it easier to find.
 
-Remember when we said that we could organize our tree in such a way that a node's metadata didn't interfere with finding its children? It turns out that not only does providing additional information about a node make it more _useful_, it also makes it easier to find.
+### Finding a Node
 
-JavaScript exposes a few ways of finding DOM nodes more or less directly, courtesy of the `document` object.
+JavaScript exposes a few ways of finding DOM nodes more or less directly,
+courtesy of the `document` object.
 
 #### `document.getElementById()`
 
-This method provides the quickest access to a node, but it requires that we know something very specific about it — its `id`. Since IDs must be unique, this method only returns one element. (If you have two elements with the same ID, this method returns the first one — keep your IDs unique!) Given the following DOM tree
+This method provides the quickest access to a node, but it requires that we know
+something very specific about it — its `id`. Since IDs must be unique, this
+method only returns one element. (If you have two elements with the same ID,
+this method returns the first one — keep your IDs unique!) Given the following
+DOM tree
 
 ``` html
 <div>
   <h5 id="greeting">Hello!</h5>
 </div>
 ```
 
-we could find the `h5` element with `document.getElementById('greeting')`. Notice how the `id` that we pass to `getElementById` is identical to the `id` in `<h5 id="greeting">`. We can assign properties to HTML nodes (or elements) simply by including them between the `<>` tags at the start of the element (so not in the `</h5>` tag, in this case).
+we could find the `h5` element with `document.getElementById('greeting')`.
+Notice how the `id` that we pass to `getElementById` is identical to the `id` in
+`<h5 id="greeting">`. We can assign properties to HTML nodes (or elements)
+simply by including them between the `<>` tags at the start of the element (so
+not in the `</h5>` tag, in this case).
 
 **Try it out!**
 
-Open up your web inspector (command+option+j on OS X) and find an element on the page — make note of its `id`. Then open up your console, type `document.getElementById('theIdYouTookNoteOf')`, and check out your handy dandy DOM node. Try changing a few of its properties!
+Open up your web inspector (command+option+j on OS X) and find an element on the
+page — make note of its `id`. Then open up your console, type
+`document.getElementById('theIdYouTookNoteOf')`, and check out your handy dandy
+DOM node. Try changing a few of its properties!
```

Comment:
Two points:

1. the HTML section of the curr calls these "attributes"
2. the "Try changing..." do they know how to do this yet?  Would we want to use the context menu to find the element and then edit it? Or is this even necessary? I don't think we've discussed using dot-notation to access methods or properties yet....I _think_ it might be best to cut?